frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts

```diff
@@ -19,7 +19,6 @@ import {createLocalInjector} from "core-app/modules/fields/edit/editing-portal/e
 import {IFieldSchema} from "core-app/modules/fields/field.base";
 import {EditFieldService, IEditFieldType} from "core-app/modules/fields/edit/edit-field.service";
 import {ResourceChangeset} from "core-app/modules/fields/changeset/resource-changeset";
-import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/modules/fields/edit/editing-portal/edit-form-portal.component.ts#L22"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>